moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -79,21 +82,13 @@ const NewProjectView = () => {
     project_priority: "",
     project_description: "",
     project_name: "",
-    start_date: "2021-01-01",
+    start_date: moment().format("YYYY-MM-DD"),
```

Comment:
Isn't moment deprecated? No problem with it, I am familiar and I like it, just wondering if we want to continue working with it. :)

Reply:
Yeah, it definitely is deprecated. I'm all about `date-fns`, `luxon`, or something else so we can learn a new library! There are only a few uses of `moment` in this repo so I don't think that would be hard at all. 👍